play-json/js/src/test/scala-3/play/api/libs/json/EnumSpec.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import ScalaTestPosition._
+
+final class EnumSpec extends AnyWordSpec with Matchers {
+  "EnumFormat" should {
+    import TestEnums.EnumWithDefaultNames._
+
+    // https://gitter.im/lampepfl/dotty?at=5ee22d1e7b6da9126a8b4a51 ¯\_(ツ)_/¯
```

Comment:
Not sure what was hiding behind this link, however with Scala 3 things now work again, see #848